examples/polyfill/platform/ARKitWrapper.js

```diff
@@ -228,10 +234,10 @@ export default class ARKitWrapper extends EventHandlerBase {
 
 	*/
 	_onWatch(data) {
-		this._rawARData = JSON.parse(data)
+		this._rawARData = data
```

Comment:
I changed the iOS app to pass the data as a JSON string to the Javascript.

On lline 285 of WebController.m I added single quotes around the parameter:
NSString *jsScript = [NSString stringWithFormat:@"%@('%@')", name, jsString];

Without those quotes, Javascript was unable to execute jsScript.